cohortextractor2/dsl.py

```diff
@@ -227,6 +285,15 @@ class Column:
 
 
 def not_null_patient_series(patient_series: PatientSeries) -> PatientSeries:
+    """
+    Removes all null values from a PatientSeries.
```

Comment:
This implies the original PatientSeries is modified in-place, contradicting `Returns`

Reply:
thaks - hopefully cleared this up